common/src/main/scala/com/gu/monitoring/products/RecurringContributionsMetrics.scala

```diff
@@ -0,0 +1,21 @@
+package com.gu.monitoring.products
+
+import com.gu.monitoring._
+import Metrics._
+class RecurringContributionsMetrics(
+  val method: String,
+  val period: String
+) extends CloudWatch(Seq(
+  productName("RecurringContributor"),
+  paymentMethod(method),
+  subscriptionPeriod(period)
+)) {
+
+  def putContributionSignUpStartProcess(): Unit = put(s"monthly-contributor-sign-up-start")
```

Comment:
This (and  `putZuoraAccountCreated `) shouldn't return Unit, it should return a Future which completes when the put request has completed.